### PR TITLE
Updated README application instructions

### PR DESCRIPTION
**Pull request by leefaisonr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,23 +27,34 @@ Startup: `rake servers:start`
...
+
+To list all import services for the application: `rake -T | grep import`
+
+To import the GuideCard records: `rake import:import_guide_cards`
+To import the SubGuideCard records: `rake import:import_sub_guide_cards`
+
+The CardImage records are the images that are included in the GuideCard and SubGuideCard records. These will take the longest to import (totaling ~1.5 million images).
 
-To import the GuideCards records: `rake import:import_guide_cards`
+To import the CardImage records: `rake import:import_card_images`
```
Comment (hackartisan) [inline README.md:57]: I suspect this requires some aws profile configuration; maybe that's worth noting here and fleshing out in the future when someone else tries to do it?

**Review by hackartisan — CHANGES_REQUESTED** [review]
maybe add a note that aws configuration will be required? If it's documentation you have easily accessible a link would be amazing, otherwise I think it's fine to leave it for now as just a heads up

File: README.md (hunk 1 above)
Comment (leefaisonr) [reply README.md:57]: Hi Anna, 

I have updated the file with the requested changes. I included some instructions on setting up aws for a local machine (instructions based on [onboarding.md](https://github.com/pulibrary/rdss-handbook/blob/2d571795873730fffc95fb8b94c3aa6fc5977d2c/onboarding.md?plain=1#L8)). Do you think I should just leave it as a note, that aws setup is required, or that these instructions are detailed enough. Thanks!

**Comment by hackartisan** [comment]
I'll adding an issue link to the description

**Pull request merged**